keystone_plugin/project.py

```diff
@@ -144,7 +149,53 @@ def update_quota(tenant_id, quota, client, what_quota):
 def delete_quota(project_id, quota, client, what_quota):
     deleting_quota = quota.get(what_quota)
     if deleting_quota:
-        if what_quota == 'neutron':
+        if what_quota == NEUTRON:
             client.delete_quota(tenant_id=project_id)
         else:
             client.quotas.delete(tenant_id=project_id)
```

Comment:
I would like to see some exception handling, either logging or raising what happened here.